infra/gcp/runbook.md

```diff
@@ -0,0 +1,29 @@
+# k8s-artifacts-prod Runbook for GCR Image issues
+
+## A bad image is detected by the auditor
+
+### Image deletions or overwrites
+
+Reinstate a known-good version of the image by re-running the promoter in
+non-dry-run mode. This can be done by manually running the `post-k8sio-cip` Prow
+job. Ask test-infra [oncall][oncall].
+
+If the manual promotion run above fails to reinstate all original images, then run:
+
+```
+$ <k8s.io_ROOT>/infra/gcp/backup_tools/restore_prod.sh
+```
+
+to restore production from the backups.
+
+## Deploy a new auditor image
```

Comment:
Is there any documentation for the auditor image?

Also, how is the auditor image related to GCR images in general?
Does the auditor send any sort of information/alert in someplace?